airbyte-container-orchestrator/src/main/java/io/airbyte/container_orchestrator/ContainerOrchestratorApp.java

```diff
@@ -212,7 +212,11 @@ private static ProcessFactory getProcessBuilderFactory(final Configs configs, fi
       // exposed)
       KubePortManagerSingleton.init(OrchestratorConstants.PORTS);
 
-      return new KubeProcessFactory(workerConfigs, configs.getJobKubeNamespace(), fabricClient, kubeHeartbeatUrl, false);
+      return new KubeProcessFactory(workerConfigs,
+          configs.getJobKubeNamespace(),
```

Comment:
curious, do you have different code styles? what is your IDE length?

Reply:
that's fine. mainly checking if this was automatic. this doesn't violate our guidelines so I think we can leave this as is.